config/toml.go

```diff
@@ -553,6 +553,17 @@ peer_query_maj23_sleep_duration = "{{ .Consensus.PeerQueryMaj23SleepDuration }}"
 # reindex events in the command-line tool.
 discard_abci_responses = {{ .Storage.DiscardABCIResponses}}
 
+# The representation of keys in the database.
```

Comment:
docs update (docs/explanation/core/configuration.md) is missing

Reply:
Done, also the part on compaction was missing so I opened a PR for it https://github.com/cometbft/cometbft/pull/2600